app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -43,65 +44,51 @@ const Notes = () => {
   }
 
   return (
-    <Container>
-      <Header
-        title="Notes"
-        actionBlock={
-          <Button
-            icon="ri-add-line"
-            label="Add new note"
-            size="small"
-            onClick={() => setShowNewNotePane(true)}
-          />
-        }
-        searchProps={{
-          value: searchTerm,
-          onChange: e => setSearchTerm(e.target.value),
-        }}
-      />
-      {notes.length ? (
-        <>
-          <SubHeader
-            rightActionBlock={
-              <Button
-                disabled={!selectedNoteIds.length}
-                icon={Delete}
-                label="Delete"
-                size="small"
-                onClick={() => setShowDeleteAlert(true)}
-              />
-            }
+    <>
+      <SideMenu showMenu={showMenuBar} />
+      <Container>
+        <Header
+          menuBarToggle={() => setShowMenuBar(!showMenuBar)}
```

Comment:
https://academy.bigbinary.com/learn-react/pages/know-when-to-use-previous-state-for-state-update

Reply:
@KrishnapriyaSkk Even if you don't use the previous state in this component, there may not be any side effects at the moment. But it's good practice to always prefer to use the prev state itself unless there is any negative impact of doing the same.

So here prefer the following:

```suggestion
menuBarToggle={() => setShowMenuBar(showMenuBar => !showMenuBar)}
```

If you feel that the suggestion comes with any negative impacts, then please lemme know.

Reply:
Yes. Thank you. I understood and  I have made that change.